mungegithub/mungers/lgtm_handler.go

```diff
@@ -118,25 +117,21 @@ func (h LGTMHandler) addLGTMIfReviewerSaidSo(obj *github.MungeObject) {
 		// See https://github.com/kubernetes/contrib/issues/1389#issuecomment-235161164
 		// TODO: An approver should be acceptable.
 		// See https://github.com/kubernetes/contrib/pull/1428#discussion_r72563935
-		if isReviewer(user, reviewers) {
+		if activeUsers.IsAssignee(user) {
 			obj.AddLabel(lgtmLabel)
 			return
 		}
 	}
 	if len(users) != 0 {
 		// if somebody that is NOT the reviewer says "LGTM", it would be nice to mention the reviewer, rather than just ignore.
 		// Note: assuming assignees are reviewers here. If not true anymore, also notify assignees.
-		obj.WriteComment(fmt.Sprintf("LGTM from users (%v). Can reviewers (%v) review this?", atUserFormat(users), atUserFormat(reviewers)))
+		obj.WriteComment(
+			fmt.Sprintf("LGTM from users (%v). Can reviewers (%v) review this?",
+				mungerutil.JoinMentions(mungerutil.GetUsersList(users)),
+				mungerutil.JoinMentions(activeUsers.Assignees())))
```

Comment:
style nit: closing parens on lines than balance out their opening paren. (you can add extra commas if need be)